### ISSUE TITLE
The crontab is set up unreliably

### ISSUE THREAD

**Issue by Rytisgit** [Rytisgit/dgamelaunch-dcss-forks-server#7]
restarting the server seems to write multiple time to the crontab, without clearing it correctly. and it doesn't seem to activate to actually pull in the updates. something to figure out.

**Comment by Rytisgit** [comment]
mentioned in readme, need to get it working still.

**Comment by refracta** [comment]
I haven't gone into the container to closely examine the issue, but setup-cron.sh doesn't seem to have any major problems at first glance. It creates the cron temporary files and deletes them properly. Could it be that `crontab -r` is not working correctly?

**Comment by Rytisgit** [comment]
I ran a few docker compose up and down cycles, and it does seem as you say regarding cron being set up properly. Perhaps the last change I made really did fix it. 

It's just something that was a pain to set up correctly initially, I experienced both the crontab being filled up with multiple copies of the scripts and cron seemingly not executing the commands, causing the binaries not being updated. If it really works now then that's great, I wasn't really tracking it in the last few months.
